src/smc-webapp/editor.coffee

```diff
@@ -1472,23 +1472,23 @@ class CodeMirrorEditor extends FileEditor
         if not @examples_dialog?
             $target = @mode_display.parent().find('.react-target')
             {render_examples_dialog} = require('./examples')
-            @examples_dialog = render_examples_dialog($target[0], @project_id, @filename, lang = @_current_mode, cb = @example_insert_handler)
+            @examples_dialog = render_examples_dialog($target[0], @project_id, @filename, lang = @_current_mode)
```

Comment:
I think please don't write code like this, because this is not how CoffeeScript works (not like python).   E.g., in CS:
```
f(a=5,b=7)
```
and 
```
f(a=7,b=5)
```
are **completely different**, whereas in python they are the same.

The code you have above will work right now sort of by accident, because lang just happens to be in the right position and assignments evaluate to the thing assigned to.   Just delete the "lang=", please.